target/linux/lantiq/files-4.19/arch/mips/boot/dts/lantiq/vr9_avm_fritz7430.dts

```diff
@@ -0,0 +1,276 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "avm,fritz7430", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 7430";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &info_red;
+		led-running = &power;
+		led-upgrade = &info_red;
+
+		led-dsl = &info_green;
```

Comment:
Is this vendor behavior?

Reply:
Not quite. I'm going to adjust that.

The question here is: In the stock firmware, the DSL status is indicated by the power LED
(blinking=no connection and vice versa). Could I just replace that by the green info LED?

Reply:
Typically, we want to preserve the LED behavior from vendor firmware, as that will be what the user expects, in contrast to some LED patterns "invented" by the patch author.

However, the blinking pattern you describe would actually be quite misleading, as the user would think his device is still booting.

In such a case, I typically recommend to set no LED (for DSL in this case) by default, and have the user configure it as he/she wishs.